mysql-test/t/ssl_san_ipv6.test

```diff
@@ -0,0 +1,39 @@
+--source include/not_embedded.inc
+--source include/have_ssl.inc
+
+# This is currently a workaround to determine if the server was compiled with
+# OpenSSL rather than YaSSL (which currently does not support TLSv1.2).
+# As YaSSL does not implement 'X509_get_ext_d2i()' function and always
+# returns NULL, '--ssl-verify-server-cert' can only check commonName ('CN')
+# entry in the certificate and therefore this test won't pass.
+--source include/have_tlsv1_2.inc
+
+--source include/count_sessions.inc
+
+# As both 'check_ipv6' and 'have_ipv4_mapped' try to establish additional
+# connections for testing, put them after 'count_sessions'
+--source include/check_ipv6.inc
+--source include/have_ipv4_mapped.inc
+
+#
+# lp:1673656 "SSL Certificate Subject ALT Names with IPs or DNS: not respected with --ssl-verify-server-cert"
+# Oracle Bug #68052 "SSL Certificate Subject ALT Names with IPs not respected with --ssl-verify-serve"
+#
+
+# This is a IPV6 companion of 'main.ssl_san' MTR test case
```

Comment:
Isn't this test also checking v4 ? If so - this comment and the test name is misleading.